packages/gatsby-source-npm-package-search/src/gatsby-node.js

```diff
@@ -0,0 +1,83 @@
+const algoliasearch = require(`algoliasearch`)
+const crypto = require(`crypto`)
+
+const client = algoliasearch(`OFCNCOG2CU`, `f54e21fa3a2a0160595bb058179bfb1e`)
+var index = client.initIndex(`npm-search`)
+
+const createContentDigest = obj =>
+  crypto
+    .createHash(`md5`)
+    .update(JSON.stringify(obj))
+    .digest(`hex`)
+
+exports.sourceNodes = async (
+  { boundActionCreators, createNodeId },
+  { keywords }
+) => {
+  const { createNode } = boundActionCreators
+
+  console.log(`Grabbing local NPM packages...`)
+
+  let buildFilter = []
+
+  keywords.forEach(keyword => {
+    buildFilter.push(`keywords:${keyword}`)
+  })
+
+  const data = await index.search({
+    query: ``,
+    filters: `(${buildFilter.join(` OR `)})`,
+    hitsPerPage: 1000,
+  })
+
+  // TODO: Currently the gatsby-transformer-remark plugin
```

Comment:
This needs fixed

Reply:
@KyleAMathews I pulled this out and found the problem packages, in terms of fixing is something like a little regular expression that replaces the badges and images with empty strings in the readme's content (so the markdown can still function) good?